.github/ISSUE_TEMPLATE/10_bug_report.yml

```diff
@@ -1,5 +1,7 @@
-name: Bug report for QField (not any other QField ecosystem product)
-description: Create a bug report to help us improve QField.
+name: Bug report for QField
```

Comment:
Thanks, I was going to ask for that.

Reply:
I did it very literal from the QGIS template :)